NEXTCLOUD.cmake

```diff
@@ -15,6 +15,34 @@ set( APPLICATION_VIRTUALFILE_SUFFIX "nextcloud" CACHE STRING "Virtual file suffi
 set( LINUX_PACKAGE_SHORTNAME "nextcloud" )
 set( LINUX_APPLICATION_ID "${APPLICATION_REV_DOMAIN}.${LINUX_PACKAGE_SHORTNAME}")
 
+option(DEV_BUILD "If enabled, name variables will be appended with DEV_SUFFIX in order to reduce conflicts." OFF)
+if (NOT DEV_SUFFIX)
+  set(DEV_SUFFIX "Dev"
+    CACHE STRING
+      "Choose a suffix with which to append build name variables to prevent conflicts."
+  FORCE)
+endif ()
+
+function(ADD_SUFFIXES SUFFIX_STRING)
+    string(TOLOWER ${SUFFIX_STRING} SUFFIX_LOWER)
+    string(APPEND APPLICATION_NAME " (${SUFFIX_STRING} Build)")
```

Comment:
I would remove the "Build" and parentheses. I don't see any value in that. "Nextcloud Daily" sounds and looks a lot better to me than "Nextcloud (Daily Build)". It's like Chrome does with it's dev/daily builds. They call them just "Chrome Canary".

Reply:
Yes, "Nextcloud Daily" sounds nice, but "Nextcloud Dev" or "Nextcloud Test" is more ambiguous. I can simplify it, but it might be good to revisit this later on.

Reply:
I decided to do a compromise where the code will strip out spaces and parentheses so that a user can optionally use them in a suffix without breaking the build. This way, I could choose `-DDEV_SUFFIX="(Test Build)"` while you could choose `-DDEV_SUFFIX="Daily"`.